news/2018-04-23-project-loved-week-of-april-22nd.md

```diff
@@ -0,0 +1,287 @@
+---
+layout: post
+title: "Project Loved: Week of April 22nd"
+date: 2018-04-23 22:00:00 +0000
+---
+
+Another Monday marks yet another [Project Loved](https://osu.ppy.sh/community/forums/120) vote! Come and vote for your favourite beatmaps nominated from all four game modes!
+
+[![](/wiki/shared/news/banners/project-loved.jpg)](https://osu.ppy.sh/community/forums/120)
+
+Hello and welcome to the second [Project Loved](https://osu.ppy.sh/community/forums/120) showcase of April 2018! This week contains several exciting picks for every game mode, and has something for both new and old players!
+
+For those of you who are unfamiliar with Project Loved or how the new system works, each mode has "captains" who pick a selection of your submissions to be put up for vote every two weeks. These chosen selections are each given their own community poll, and any beatmap with an approval rating above 75% (or 80% for osu!mania) after a week's time is moved into the Loved category. Beatmaps that are Loved have leaderboards, but do not award performance points (much like Qualified maps).
+
+If you'd like to vote for any of the beatmaps in this week's selection, just click the map's header image and it'll lead you to the poll!
+
+If you'd like to help out and submit beatmaps *you* love to be considered for the Loved category, please use this [submission form](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)!
+
+Once again, [to the beat!](https://www.youtube.com/channel/UCIFsWU-grp6UgGgMm57FiGQ) has created a video for each game mode showcasing our beatmap selections, so if you'd like to see the entries in action, be sure to check out the videos below!
+
+Let's get started!
+
+### Navigation
+
+- [osu!](#osu)
+- [osu!taiko](#osutaiko)
+- [osu!catch](#osucatch)
+- [osu!mania](#osumania)
+
+## <a name="osu" id="osu"></a>osu!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/35HhNAW6pTo?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+---
+
+[![](/wiki/shared/news/2018-04-23-project-loved-week-of-april-22nd/osu/revolutionary-etude.jpg)](https://osu.ppy.sh/community/forums/topics/735985)
+
+[Chopin - Revolutionary Etude](https://osu.ppy.sh/beatmapsets/539300) by [Louis Cyphre](https://osu.ppy.sh/users/186243)  
+*written by [Toy](https://osu.ppy.sh/users/2757689)*
+
+One of the most infamous 9\* maps out there, [Louis Cyphre](https://osu.ppy.sh/users/186243)'s extra difficulty originally found in the description of his ranked set of Revolutionary Etude quickly rose to fame as one of the most popular "impossible" maps on the market. High profile players [Azer](https://osu.ppy.sh/users/2155578) and [Cookiezi](https://osu.ppy.sh/users/124493) also helped boost its popularity through stream highlights of plays on the map. The erratic, geometrically patterned jumble of circles and sliders has since become one of the game's most acclaimed aim tests.
```

Comment:
https://osu.ppy.sh/beatmapsets/66350#osu/193647 ranked set of revolutionary etude